src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/xlf/AssemblyReference.xaml.zh-Hant.xlf

```diff
@@ -1,97 +1,97 @@
-      <trans-unit id="StringProperty|Identity|Description">
-        <source>Security identity of the referenced assembly (see System.Reflection.Assembly.Evidence or System.Security.Policy.Evidence).</source>
-        <target state="translated">參考組件的安全性識別 (請參閱 System.Reflection.Assembly.Evidence 或 System.Security.Policy.Evidence)。</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|ResolvedPath|DisplayName">
-        <source>Path</source>
-        <target state="translated">路徑</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|ResolvedPath|Description">
-        <source>Location of the file being referenced.</source>
-        <target state="translated">正被參考之檔案的位置。</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|Version|DisplayName">
-        <source>Version</source>
-        <target state="translated">版本</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|Version|Description">
-        <source>Version of reference.</source>
-        <target state="translated">參考的版本。</target>
-        <note />
-      </trans-unit>
-    </body>
-  </file>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="zh-Hant" original="../AssemblyReference.xaml">
+    <body>
+      <trans-unit id="BoolProperty|TreatAsUsed|Description">
+        <source>Indicates that this reference should be treated as if it were used.</source>
+        <target state="translated">表示應將此參考視為已使用。</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="BoolProperty|TreatAsUsed|DisplayName">
+        <source>Treat as used</source>
+        <target state="translated">視為已使用</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Rule|AssemblyReference|DisplayName">
+        <source>Assembly Reference</source>
+        <target state="translated">組件參考</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Rule|AssemblyReference|Description">
+        <source>Assembly Reference Properties</source>
+        <target state="translated">組件參考屬性</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="StringListProperty|Aliases|DisplayName">
+        <source>Aliases</source>
+        <target state="translated">別名</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="StringListProperty|Aliases|Description">
+        <source>A comma-delimited list of aliases to this reference.</source>
+        <target state="translated">這個參考的逗號分隔別名清單。</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="BoolProperty|Private|DisplayName">
+        <source>Copy Local</source>
+        <target state="new">Copy Local</target>

```

Comment:
This looks wrong. If you disable whitespace changes in the diff you can see that this PR just undoes a bunch of translations. It's unclear why it's doing that.

Reply:
![image](https://user-images.githubusercontent.com/350947/151881170-264e5e20-0c92-4203-8438-13a9d5ccdaae.png)